tokens/staking/stake.md

```diff
@@ -30,7 +30,7 @@ Token holders that want to easily stake their tokens can use the [Moonbeam dApp]
     | Maximum delegators per candidates |  |                                                    {{ networks.moonriver.staking.max_del_per_can }}                                                     |
```

Comment:
Max delegators per candidate has increased from 100 to 500 on Moonriver and Moonbase Alpha

Reply:
https://github.com/PureStake/moonbeam/pull/1096